### DISCUSSION TITLE
O que é preciso para se considerar Júnior no Front-end

### DISCUSSION THREAD

**Discussion by JhonnyO3** [frontendbr/forum#2310] [Discussão]
Pessoal, estava refletindo ultimamente e me encontro um pouco perdido em atribuir quando serei considerado um Júnior no mercado.. estudo front-end a +- uns 6 meses na FIAP e por conta própria e me encontrei na dúvida..

**Comment by felquis** [comment]
Fala Jhonny, tudo bem?

Não tenho tanta participação nas ultimas tendencias de junior do mercado, mas...

> perdido em atribuir quando serei considerado um Júnior no mercado

Vai depender da vaga que você preencher, se você preencher uma vaga junior você será junior, se você preencher uma vaga pleno você será pleno, mesmo que você não se defina com esses termos. Você pode ser pleno na empresa, mas vc pode se considerar junior, e você pode ser junior na empresa mas se considerar pleno. 

> estudo front-end a +- uns 6 meses na FIAP e por conta própria

Você concorda comigo que você está iniciando sua jornada? Se você se assumir como junior te fazer se sentir melhor, vá em frente e se assuma junior. Ninguém irá te julgar por isso. Mesmo que você não esteja preenchendo uma vaga junior em alguma empresa.

Mas na minha visão, junior/pleno/senior só é útil para empresas organizarem folha de pagamento. Então o iniciante se torna junior  quando está na folha de pagamento de alguma empresa como junior.

Eu mesmo na minha jornada nunca escolhi o termo junior para mim, mas depois de algum tempo apareceu pleno na minha CLT, e até senior eu acho. Mas eu me considero uma ameba. Teve momento que meu empregador disse que eu era Front-End Engineer, sendo que nunca me defini como sendo um Front-End Engineer. Enquanto outras pessoas se agarram a este termo com unha e dente.

**Reply by JhonnyO3** [reply]
Nossa, bacana este pensamento, eu me encontrava bem perdido em relação a isso! Agradeço o comentário e a paciência!
